src/administration/integrations/notifications.md

```diff
@@ -24,9 +26,41 @@ Health notifications can be set up via the [Platform.sh CLI](/gettingstarted/cli
 
 ### Email notifications
 
+#### Default email notifications
+
+When your project is new or when your project doesn't have a health notification set up, Platform.sh creates a default low-disk email notification for all [Project Admins](/administration/users.md).
```

Comment:
Will we be adding admin notifications to all notification-free projects going forward, or is it just a one-time thing for existing projects?  Viz, if I create a project in 4 months and deliberately delete the notification, will it come back?

The text here as written suggests it would, which is not what I expected (and is probably unwise).